### ISSUE TITLE
Handling 'Raises' sections where the error comes from a package

### ISSUE THREAD

**Issue by Gabriel-p** [jsh9/pydoclint#157]
For example, this dostring:

```
def some_func():
    """
    Raises
    ------
    FileNotFoundError
        If the specified file does not exist.
    pd.errors.EmptyDataError
        If the file is empty.
    KeyError
        If the required columns 'label' and 'text' are not present in the file.
    """
    df_class = pd.read_csv("my_file.csv")
```

fails with:

```
DOC502: Function `some_func` has a "Raises" section in the docstring, but there are not "raise" statements in the body
```

But `pandas` does raise those errors. Should the section be removed from the docstring?

**Comment by jonyscathe** [comment]
You should only document errors that are actually raised within that function body in that docstring.
Surely you wouldn't expect any other function that calls `some_func` to then also document the FileNotFoundError, pd.errors.EmptyDataError and KeyError in its docstring?

**Comment by Gabriel-p** [comment]
Makes sense. Thank you!
